packages/jellyfish-api-core/src/category/loan.ts

```diff
@@ -378,6 +378,24 @@ export class Loan {
       'number'
     )
   }
+
+  /**
+   * List all available auctions.
+   *
+   * @param {AuctionPagination} pagination
+   * @param {AuctionPaginationStart} [pagination.start]
+   * @param {string} [pagination.start.vaultId]
+   * @param {number} [pagination.start.height]
+   * @param {boolean} [pagination.including_start]
```

Comment:
thinking will remove `including_start` and do optional on height in future :thinking:

Reply:
You mean in the C++ side, they should not have including_start right?

Reply:
we have `including_start` very early.. so just leave it for now

Reply:
I won't add this to whale anyway.

Reply:
But, the funny thing is listAuctionHistory RPC does not have including_start. LOL